application/holder/src/service/bridge/file/select.ts

```diff
@@ -4,22 +4,32 @@ import { electron } from '@service/electron';
 import { File } from 'platform/types/files';
 import { FileType } from 'platform/types/observe/types/file';
 import { getFileEntities } from '@env/fs';
+import { scope } from "platform/env/scope";
 
 import * as Requests from 'platform/ipc/request';
 
 function any(ext?: string): Promise<File[]> {
+    const logger = scope.getLogger('SelectFile');
```

Comment:
Creating of logger is not much, but still costs something... let's create in `catch` section. I hope it will be used not very often ;)